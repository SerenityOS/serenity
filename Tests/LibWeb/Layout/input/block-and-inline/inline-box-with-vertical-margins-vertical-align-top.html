<style>
    body {
        font-family: 'SerenitySans';
    }

    #inline-box {
        background-color: red;
        margin-top: 50px;
        margin-bottom: 25px;
        width: 100px;
        height: 100px;

        display: inline-block;
        vertical-align: top;
    }
</style>Well, <div id="inline-box"></div> friends.