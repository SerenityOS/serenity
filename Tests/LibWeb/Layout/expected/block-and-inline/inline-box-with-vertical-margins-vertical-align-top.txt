Viewport <#document> at (0,0) content-size 800x600 children: not-inline
  BlockContainer <html> at (0,0) content-size 800x600 [BFC] children: not-inline
    BlockContainer <body> at (8,8) content-size 784x175 children: inline
      line 0 width: 210.828125, height: 175, bottom: 175, baseline: 13.53125
        frag 0 from TextNode start: 0, length: 6, rect: [8,8 43.125x17.46875]
          "Well, "
        frag 1 from BlockContainer start: 0, length: 0, rect: [51,58 100x100]
        frag 2 from TextNode start: 0, length: 9, rect: [151,8 67.703125x17.46875]
          " friends."
      TextNode <#text>
      BlockContainer <div#inline-box> at (51,58) content-size 100x100 inline-block [BFC] children: not-inline
      TextNode <#text>

PaintableWithLines (Viewport<#document>) [0,0 800x600]
  PaintableWithLines (BlockContainer<HTML>) [0,0 800x600]
    PaintableWithLines (BlockContainer<BODY>) [8,8 784x175]
      TextPaintable (TextNode<#text>)
      PaintableWithLines (BlockContainer<DIV>#inline-box) [51,58 100x100]
      TextPaintable (TextNode<#text>)
